Userland/Applications/Presenter/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+serenity_component(
+    Presenter
+    RECOMMENDED
+    TARGETS Presenter
+    DEPENDS ImageDecoder FileSystemAccessServer
+)
+
+
+set(SOURCES
+    main.cpp
+	PresenterWidget.cpp
+	Presentation.cpp
```

Comment:
Are those tabs?
Also :red_circle: :penguin:

Reply:
(not complaining, just explaining:) When you copy a file's contents, VSCode is not always smart enough to pick up the indentation mode, and I don't usually close and re-open files. It only doesn't happen for C++ files as I run those through the formatter with a key combo often enough.